internals/resource-managers/output.go

```diff
@@ -143,33 +143,32 @@ func (o OutputManager) CleanUp(capp rcsv1alpha1.Capp) error {
 // It returns an error if any operation fails.
 func (o OutputManager) CreateOrUpdateObject(capp rcsv1alpha1.Capp) error {
 	outputName := capp.GetName() + "-output"
-	logger := log.FromContext(o.Ctx).WithValues("CappName", capp.Name, "CappNamespace", capp.Namespace, "outputName", outputName)
+	logger := o.Log
```

Comment:
Add `WithValues` here to include the `outputName` so we don't need to mention it in the logger